ui/css/intermittent-failures.css

```diff
@@ -35,6 +35,56 @@ h1,
   display: initial;
 }
 
+/* Graph Alternate View - Table */
+.alternate-table {
+  max-width: 660px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.alternate-table .rt-th {
+  font-weight: bold;
+  font-size: 1.2rem;
+}
+
+/* Failures Count per Push Table */
+.failure-per-count .rt-th:last-child {
+  background-color: rgb(221, 102, 2, 0.8);
+}
+
+.failure-per-count .rt-tr .rt-td:last-child {
+  background-color: rgba(221, 102, 2, 0.3);
+}
+
+.failure-per-count .rt-tr.-odd .rt-td:last-child {
+  background-color: rgba(221, 102, 2, 0.5);
+}
+
+/* Failure Count vs Push Count Table */
```

Comment:
Are all of these styles necessary with the new table row color change?

Reply:
noo! I didn't send a push yet. I've sent the screenshot for approval. I will send a new push now.